src/ui/zcl_abapgit_gui_page_data.clas.abap

```diff
@@ -1,3 +1,8 @@
+*----------------------------------------------------------------------*
```

Comment:
please remove the comments added by pretty print (we don't have these anywhere else in abapGit). thanks

Reply:
ok sorry. wasn't aware of this. Removed them now :-)